docs/source/api/result_object.rst

```diff
@@ -1,12 +1,12 @@
 .. _ref_api_result_object:
 
-*************
-Result Object
-*************
+**********
+``Result``
+**********
```

Comment:
```suggestion
****************
``Result`` object
****************
```

Reply:
It's a class definition, keeping result.